pkg/subctl/cmd/gather/logs.go

```diff
@@ -120,3 +124,39 @@ func findPods(clientSet kubernetes.Interface, byLabelSelector string) (*corev1.P
 
 	return pods, nil
 }
+
+func getPreviousLogs(pod *corev1.Pod, podLogOptions *corev1.PodLogOptions, info Info) error {
+	podLogOptions.Previous = true
+	logRequest := info.ClientSet.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, podLogOptions)
+	logStream, _ := logRequest.Stream()
+
+	// TODO: Check for error other than "no previous pods found"
+
+	// if no previous pods found, logstream == nil, ignore it
+	if logStream != nil {
+		info.Status.QueueWarningMessage(fmt.Sprintf("found previous instances of pod %s", pod.Name))
+		err := writeLogs(logStream, info, pod.Name, ".log.old")
```

Comment:
"_.old_" to me kind of implies a backup of another file.  I suggest "_.log.prev_".